libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-lookup/cell-editor-lookup.component.html

```diff
@@ -1,11 +1,17 @@
+<label class="sky-screen-reader-only" skyId #labelElement="skyId">{{
+  skyComponentProperties?.ariaLabel ||
```

Comment:
Could you instead assign this value to the former `ariaLabel` on the lookup component, below?

Reply:
Updated to use the `ariaLabel` inputs.

Reply:
When using `ariaLabel` and not `ariaLabelledby`, there was an issue that the combobox did not have a label. I've updated lookup to fix that issue.